gatsby-node.ts

```diff
@@ -148,6 +148,13 @@ export const createPages: GatsbyNode["createPages"] = async ({
 }) => {
   const { createPage, createRedirect } = actions
 
+  createRedirect({
+    fromPath: `/conf/attendee/*`,
+    toPath: `https://og-image.the-guild.dev/*`,
```

Comment:
```suggestion
    toPath: `https://og-image.the-guild.dev/conf/*`
```
can we redirect instead?